### ISSUE TITLE
Loading a trained Catboostregressor leads to crash in prediction

### ISSUE THREAD

**Issue by AlexanderZender** [alteryx/evalml#4025]
When loading a previously trained pipeline with a catboostregressor, the predict_model functionality doesn't work as expected and crash, expecting a setup run first.

![image](https://user-images.githubusercontent.com/79153884/220596228-da0554d3-9f0d-4d62-95f3-e7141537133c.png)

**Comment by jeremyliweishih** [comment]
Thanks for filing @AlexanderZender ! Could you let me know which Catboost version you ran when saving the pipeline and which version you ran when loading?

**Comment by AlexanderZender** [comment]
> Thanks for filing @AlexanderZender ! Could you let me know which Catboost version you ran when saving the pipeline and which version you ran when loading?

Yes, for creating and loading the pipeline Catboost 1.1.1, the current release was used
